dev/com.ibm.ws.app.manager.wab/src/com/ibm/ws/app/manager/wab/internal/WAB.java

```diff
@@ -76,6 +76,8 @@
     private final Version wabBundleVersion;
     private final String wabContextPath;
     private final WABInstaller installer;
+    private final String rawVirtualHost;
+    private String resolvedVirtualHost;
```

Comment:
I think resolveVirtualHost can be final.  At least I don't see where you modify it outside the constructor.  My understanding is that a new WAB will be create because of a restart on the bundle so it will pick up the new resolved host then.